scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -547,6 +538,25 @@ synchronized void checkChangeJobPriority(JobId jobId, JobPriority priority) thro
         }
     }
 
+    synchronized void checkPermissions(String methodName, IdentifiedJob identifiedJob, String errorMessage)
+            throws NotConnectedException, UnknownJobException, PermissionException {
+        try {
+            checkJobOwner(methodName, identifiedJob, errorMessage);
```

Comment:
as this method is now called only at one place, I think it's not necessary any more to handle it with exceptions.